packages/Testing/PHPUnit/AbstractTestCase.php

```diff
@@ -77,4 +77,9 @@ private function createConfigsHash(array $configFileInfos): string
 
         return $configHash;
     }
+
+    private function isRunningOnCI(): bool {
```

Comment:
it seems needs cs fix, you can run:

```bash
composer fix-cs -- packages/Testing/PHPUnit/AbstractTestCase.php
```

to resolve it.

Reply:
good hint - done. thx